initial commit

Beablad opened the pull request.

Commit:
initial commit

Commit:
initial commit

Commit:
initial commit

Commit:
initial commit

Commit:
feat: добавлен функционал создания фильмов и пользоваетелей через rest контролеры

Commit:
feat: добавлен функционал создания фильмов и пользоваетелей через rest контролеры

Troitsa commented on src/main/java/ru/yandex/practicum/filmorate/LocalDateTimeAdapter.java:
```diff
@@ -0,0 +1,27 @@
+package ru.yandex.practicum.filmorate;
```
желательно класс разместить в какой-нибудь папке util, а не в корне проекта

Troitsa commented on src/main/java/ru/yandex/practicum/filmorate/LocalDateTimeAdapter.java:
```diff
@@ -0,0 +1,27 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateTimeAdapter extends TypeAdapter<LocalDate> {
```
LocalDateAdapter?

Troitsa commented on src/main/java/ru/yandex/practicum/filmorate/controllers/FilmController.java:
```diff
@@ -0,0 +1,75 @@
+    private final int MAX_CHAR_DESCRIPTION = 200;
+    private final LocalDate MIN_RELEASE_DATE = LocalDate.of(1895, 12, 28);
+    private int id = 1;
+    Map<Integer, Film> filmList = new HashMap<>();
```
модификатор доступа упущен

Troitsa commented on src/main/java/ru/yandex/practicum/filmorate/controllers/UserController.java:
```diff
@@ -0,0 +1,77 @@
+@RequestMapping(value = "/users")
+public class UserController {
+
+    Map<Integer, User> userList = new HashMap();
```
Нарушение инкапсуляции - модификаторы доступа упущены

Commit:
fix: LocalDateTimeAdapter.java переименован в LocalDateAdapter.java и перенесен в папку Util
добавлены модификаторы доступа private в классы FilmController.java и UserController.java
